x-pack/plugins/infra/public/utils/kuery.ts

```diff
@@ -10,7 +10,8 @@ import { esKuery } from '../../../../../src/plugins/data/public';
 
 export const convertKueryToElasticSearchQuery = (
   kueryExpression: string,
-  indexPattern: DataViewBase
+  indexPattern: DataViewBase,
+  swallowErrors: boolean = true
```

Comment:
Maybe we can convert this to a small helper instead and use composition for the same effect?
I think it's more clear at call site to read something like this instead:
`tryOrFallback(convertKueryToElasticSearchQuery(kuery, indexPatterns), '')`
This leaves the places that should throw more clear also by not sending the magic boolean false into them.